[TESTS] karma-mocha-reporter doesn't print object diff

While I'm testing some issues in the engine I get in trouble with the karma output, which is pretty annoying sometimes, especially in deep comparisons:
```
AssertionError: expected { Object (parent, _attrs, ...) } to deeply equal { Object (parent, _attrs, ...) }
```

So I propose to add the option:
```
mochaReporter: {
    showDiff: true
}
```
 to Karma config as a flag or permanently.

--- a/packages/ckeditor5-dev-tests/lib/utils/automated-tests/getkarmaconfig.js
+++ b/packages/ckeditor5-dev-tests/lib/utils/automated-tests/getkarmaconfig.js
@@ -126,7 +126,12 @@ module.exports = function getKarmaConfig( options ) {
 		concurrency: Infinity,
 
 		// How long will Karma wait for a message from a browser before disconnecting from it (in ms).
-		browserNoActivityTimeout: 0
+		browserNoActivityTimeout: 0,
+
+		// Shows differences in object comparison.
+		mochaReporter: {
+			showDiff: true
+		}
 	};
 
 	if ( process.env.TRAVIS ) {

--- a/packages/ckeditor5-dev-tests/tests/utils/automated-tests/getkarmaconfig.js
+++ b/packages/ckeditor5-dev-tests/tests/utils/automated-tests/getkarmaconfig.js
@@ -94,7 +94,10 @@ describe( 'getKarmaConfig', () => {
 			},
 			singleRun: true,
 			concurrency: Infinity,
-			browserNoActivityTimeout: 0
+			browserNoActivityTimeout: 0,
+			mochaReporter: {
+				showDiff: true
+			}
 		} );
 	} );
 

